query/lambdas/api_server/v1/endpoints.py

```diff
@@ -20,6 +27,16 @@ def query(query_string):
     return {'query': query_string, "results": formatted_query_results}, requests.codes.ok
 
 
+@return_exceptions_as_http_errors
+def webhook(new_data):
+    new_data = json.loads(new_data, strict=False)
+    response = sqs_client.send_message(
+        QueueUrl=f"https://sqs.us-east-1.amazonaws.com/861229788715/dcp-query-data-input-queue-{Config.deployment_stage}",
```

Comment:
1. The `account_id` here should be parameterized.
2. It would probably be cleaner to parameterize the whole queue name and have it specified in the environment and then load it into all the lambdas you need. Rationale: if you changed the name here, you'd have to track down all the other places in the repo where the name is used and change it in the same way.

Reply:
👍